auth2client/src/App.css

```diff
@@ -0,0 +1,32 @@
+.App {
+  text-align: center;
```

Comment:
This boilerplate CSS can be deleted since you are no longer using the original create-react-app logo and header